CHANGE_LOG.md

```diff
@@ -11,6 +11,10 @@ All notable changes to this project will be documented in this file.
 - Fixed an issue, where after the agent end the chat and C2 sees the disconnect banner after toggling, refreshing the browser does not show the message box.
 - Clear ChatSDK's internal `liveChatContext` when `conversationState` is set to `Closed` on `startChat()`
 
+### Changed
+
+- Updated a few exception's details on chat start
```

Comment:
Can we be more specific with this change log description? Perhaps mentioning the specific exceptions being covered?